src/pages/SubmissionHistory/SubmissionHistory.js

```diff
@@ -123,16 +134,35 @@ export const SubmissionHistory = () => {
                 paddingTop: '1em'
               }),
               columns: [
-                ...isActionMenuEnabled() ?
+                ...getConfig().isActionMenuEnabled ?
                   [{
                     size: { basis: 100, grow: 0 },
                     field: 'actions',
-                    headerRenderer: () => h(TextCell, ['Actions']),
-                    cellRenderer: () => {
-                      return div(
-                        { style: { textAlign: 'center' } },
-                        [icon('cardMenuIcon', { size: 24, onClick: () => { window.alert('TODO: go to actions menu') } })]
-                      )
+                    headerRenderer: () => h(TextCell, {}, ['Actions']),
+                    cellRenderer: ({ rowIndex }) => {
+                      return h(MenuTrigger, {
+                        'aria-label': 'Action selection menu',
+                        popupProps: {
+                          style: { left: '-20px' }
+                        },
+                        content: h(Fragment, [
+                          h(MenuButton, {
+                            style: { fontSize: 15 },
+                            onClick: () => {
+                              if (isRunSetInTerminalState(paginatedPreviousRunSets[rowIndex].state) === true) {
+                                notify('error', 'Error aborting run set', { detail: 'Cannot abort a submission that is in a terminal state.' })
```

Comment:
I think instead of showing the error we should disable the Abort button with a tooltip saying something along the lines of `...this submission is already in terminal state`. This would align it with what we have in Terra UI

Reply:
You might be able to use the same `isRunSetInTerminalState` for disabling the button

Reply:
I forgot you gave an example error message here, but let me know if the one I have in the revision is good enough.